carservice/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.core.validators import MaxValueValidator
+
+
+class Car(models.Model):
+    serial_number = models.CharField(max_length=17)
+    car_mileage = models.PositiveIntegerField(validators=[MaxValueValidator(300000)])
```

Comment:
to dość dziwne zastrzeżenie, bo co jeśli coś takiego by się stało?

Reply:
Nie do końca rozumiem